docs/source/user/vectorize.rst

```diff
@@ -168,10 +168,9 @@ There are two things there:
   only support ``int64`` arrays.
 
 .. note::
-   The concrete signature does not allow for scalar values, even though
-   the layout may mention them.  In this example, the second argument is
-   declared as ``int64[:]``, not ``int64``.
-   This is why it must be dereferenced by fetching ``y[0]``.
+   1D array type can also receive scalar arguments (those with shape ``()``).
+   In the above example, the second argument can be declared as
```

Comment:
Rephrase to "In the above example, the second argument also could be declared as"